src/Controller/V2/Config/ConfigSchema.ts

```diff
@@ -0,0 +1,45 @@
+/*!
+ * © 2023 Atypon Systems LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Joi from 'joi'
+
+import { jsonHeadersSchema } from '../../BaseSchema'
+
+export const configSchema: Joi.SchemaMap = {
+  headers: jsonHeadersSchema.headers,
+}
```

Comment:
Shouldn't we add a query object here with id? 
And shouldn't we also make all ids inside different queries required? 
@mnatsheh

Reply:
@mnatsheh The id query to the `configSchema` has been added by the this commit [Update schemas headers](https://github.com/Atypon-OpenSource/manuscripts-api/pull/62/commits/88fe0643636456bad906578f1bd70a4b9c392ecf), which was added  after this comment. 
I'll make the id required and add the regex.

Reply:
Got it. Thanks.